Sources/AcknowLocalization.swift

```diff
@@ -45,6 +45,24 @@ open class AcknowLocalization {
         return Locale.preferredLanguages.first
     }
 
+    /**
+     Returns the resources bundle (for localizations).
+
+     - returns: The resources bundle.
+     */
+    class func resourcesBundle() -> Bundle? {
+        #if SWIFT_PACKAGE
```

Comment:
Where does this `SWIFT_PACKAGE` value come from? The last time I tried to add SPM support, there was no such mechanism to dynamically check whether the `Bundle.module` object was available or not.